src/video_core/renderer_vulkan/liverpool_to_vk.h

```diff
@@ -76,6 +76,12 @@ static inline vk::Format PromoteFormatToDepth(vk::Format fmt) {
         return vk::Format::eD32Sfloat;
     } else if (fmt == vk::Format::eR16Unorm) {
         return vk::Format::eD16Unorm;
+    } else if (fmt == vk::Format::eR8G8B8A8Unorm) {
+        return vk::Format::eR32Uint;
+    } else if (fmt == vk::Format::eR8G8B8A8Srgb) {
+        return vk::Format::eR32Uint;
+    } else {
+        LOG_ERROR(Render_Vulkan, "Unexpected Depth Format {}", vk::to_string(fmt));
```

Comment:
You don't need a separate log line for this, you can just chnage the unreachable below to ```UNREACHABLE_MSG("Unexpected Depth Format {}", vk::to_string(fmt));```